gen_import_utils.py

```diff
@@ -55,19 +83,15 @@ def extract_last_folders(path, number: int):
 
```

Comment:
no longer pulling images from nested date paths, so modified to use simple date input and config prefix.